### PR TITLE
Furasian Fluff item

### PR DESCRIPTION
**Pull request by Aurorablade**
Fluff item: Coat, Navy, Red and black colors
ckey: Furasian
Character: Fillmoore Grayson
Head approved: yes.

:cl:
rscadd: Vet Coat Fluff item.
tweak: Spacepod mod kit now uses kit blueprints and not KITE blueprints (spelling error).
/:cl:

### CODE REVIEW

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -1348,4 +1348,20 @@
 	righthand_file = 'icons/mob/inhands/fluff_righthand.dmi'
 	icon_state = "teri_horn"
 	item_state = "teri_horn"
-	honk_sound = 'sound/items/teri_horn.ogg'
\ No newline at end of file
+	honk_sound = 'sound/items/teri_horn.ogg'
+
+/obj/item/clothing/suit/fluff/vetcoat //Furasian: Fillmoore Grayson
+    name = "Veteran Coat"
+    desc = "A old, yet well-kept Nanotrasen uniform. Very few of its kind are still produced."
```
Comment (Tayyyyyyy) [inline code/modules/customitems/item_defines.dm]: Should this say "An old", unless this is intentional?

Comment (FalseIncarnate) [reply code/modules/customitems/item_defines.dm]: "An" would be correct in that case (use an if the following word begins with a vowel is general rule of thumb)

File: code/modules/customitems/item_defines.dm [hunk 2]
```diff
@@ -1348,4 +1348,20 @@
...
+    icon = 'icons/obj/custom_items.dmi'
+    icon_state = "alchemistcoatblack"
```
Comment (Fox-McCloud) [inline code/modules/customitems/item_defines.dm]: These are all spaces; not tabs.

File: code/modules/customitems/item_defines.dm [hunk 3]
```diff
@@ -1348,4 +1348,20 @@
...
-	honk_sound = 'sound/items/teri_horn.ogg'
\ No newline at end of file
+	honk_sound = 'sound/items/teri_horn.ogg'
+
+/obj/item/clothing/suit/fluff/vetcoat //Furasian: Fillmoore Grayson
+	name = "Veteran Coat"
+	desc = "An old, yet well-kept Nanotrasen uniform. Very few of its kind are still produced."
+	icon = 'icons/obj/custom_items.dmi'
+	icon_state = "alchemistcoatblack"
+	item_state = "alchemistcoatblack"
```
Comment (Fox-McCloud) [inline code/modules/customitems/item_defines.dm]: Did you test this out to make sure it worked? And to make sure the custom in-hands work?

**Pull request merged**